### PR TITLE
[ADLP] Update FSP/VBT for MR5 release

### PR DESCRIPTION
**Pull request by vincentchchen**
- update FSP version to IoT ADL-P MR5 (0C.01.B0.50)
- update VBT version to 251
- Update platform version to 1.5

### CODE REVIEW

**Comment by vincentchchen** [comment]
The QEMU check failed due to the added VbtAdlP.json contains Tab characters.
The VbtAdlP.json file is copied as is, so it is not modified against the errors.

File: Platform/AlderlakeBoardPkg/VbtBin/VbtAdlP.dat
Comment (gdong1) [inline Platform/AlderlakeBoardPkg/VbtBin/VbtAdlP.dat:1]: Is this file same with the one from here https://github.com/intel/FSP/tree/master/AlderLakeFspBinPkg/IoT/AlderLakeP/VBT?
Maybe we should get from there if it is same. (Same as FSP),

Comment (vincentchchen) [reply Platform/AlderlakeBoardPkg/VbtBin/VbtAdlP.dat:1]: Yes, they are the same. I have updated this PR to copy the JSON file from FSP repo.
Previously, sometimes, the FSP repo didn't update VBT files along with FSP files, so we may not get the latest one, especially  not easy to manage for different ADL SKUs.
Anyway, we can take extra care of this VBT part when FSP files are being updated for new MRs.

Comment (gdong1) [reply Platform/AlderlakeBoardPkg/VbtBin/VbtAdlP.dat:1]: ok. looks good to me.

**Review by gdong1 — APPROVED** [review]

**Pull request merged**